backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5753110_BusinessRule_C_OrderLine_FullLURequired_WithFunction.sql

```diff
@@ -0,0 +1,28 @@
+-- Name: C_OrderLine Require Full LU
+-- 2025-04-29T20:35:20.696Z
+UPDATE AD_Val_Rule SET Code=' /* partner does not require a full LU */
```

Comment:
qtyenteredTU is nullable (and based on `de.metas.adempiere.gui.search.impl.HUPackingAwareBL#calculateQtyTU` null is ok on several scenarios). Should we also exclude those cases from throwing up a validation error?